tests/backup/backup_helper.go

```diff
@@ -2072,3 +2073,14 @@ func FetchNamespacesFromBackup(ctx context.Context, backupName string, orgID str
 	backedUpNamespaces = resp.GetBackup().GetNamespaces()
 	return backedUpNamespaces, err
 }
+
+// RemoveElementByValue remove the given value from the array.
+func RemoveElementByValue(arr interface{}, value interface{}) {
```

Comment:
You can specify the comment as:
`// RemoveElementByValue remove the given value from the array. Pass a pointer to the array and the element by value. This function will panic if passed incorrect types.`.
Also, if you don't want this function to panic, and want it to simply fail, you can add a `defer func(){r:=recover()}()` to it.  But then you won't know if it failed, unless you print the error

Reply:
Since the function will be used specifically at the testcase level, there is less chance for anyone to pass a wrong value here. I will make the description more descriptive